core/domain/topic_services_test.py

```diff
@@ -147,6 +147,29 @@ def test_update_topic(self):
         self.assertEqual(topic_summary.version, 3)
 
 
+    def test_delete_skill(self):
+        topic_services.delete_skill(
```

Comment:
In all these it might be worth validating that the correct changelist model was saved? (if we don't do that in other tests?)

Reply:
Done.